jetstream/tas_api.py

```diff
@@ -30,10 +28,16 @@ def tacc_api_get(url, username=None, password=None):
     if not password:
         password = settings.TACC_API_PASS
     logger.debug('url: %s', url)
-    resp = requests.get(
-        url,
-        auth=(username, password),
-        timeout=settings.TACC_API_TIMEOUT)
+    try:
+        resp = requests.get(
+            url,
+            auth=(username, password),
+            timeout=settings.TACC_READ_API_TIMEOUT)
+    except ReadTimeout:
+        raise TASAPIException(
+            "TAS API is taking too long to respond, we're timing out ({})".
+            format(url)
+        )
```

Comment:
It's not clear to me if this is the best place to catch this exception. Open to ideas.